add option for EventAwareLumiBase to observe a 47h wall time estimate cap

hufnagel opened the pull request:
The Tier0 has very unreliable timePerEvent and sizePerEvent estimates as running conditions can change greatly, leading to huge differences within the same dataset from run to run. To not create jobs that can never run, cap the wall time and disk estimates.

hufnagel commented:
Ok, we know the PromptReco spec is partially broken in the way it handles skims. This will be cleaned up at some point, can be ignored for now.

Why do I get so many bogus error messages for REST unit tests though ? The code changed here has nothing to do with these unit test. @ericvaandering ?

PerilousApricot commented:
The rest stuff is a constant thorn. It's super finnicky
>
>

-- 
Sent from Gmail Mobile

ericvaandering commented:
These tests seem to be flipping between OK and not OK. I haven’t had a chance to look at them.


On Jul 28, 2015, at 8:07 PM, Dirk Hufnagel <notifications@github.com<mailto:notifications@github.com>> wrote:


Ok, we know the PromptReco spec is partially broken in the way it handles skims. This will be cleaned up at some point, can be ignored for now.

Why do I get so many bogus error messages for REST unit tests though ? The code changed here has nothing to do with these unit test. @ericvaandering<https://github.com/ericvaandering> ?

—
Reply to this email directly or view it on GitHub<https://github.com/dmwm/WMCore/pull/6077#issuecomment-125794839>.

ericvaandering commented:
Jenkins should now realize that scores are floats, not strings. So going to 

retest this please

ericvaandering commented:
test this please

hufnagel commented:
Ok, I need to change this, should get retested once I push the modified commit.

hufnagel committed:
add option for EventAwareLumiBase wall time and disk caps, use them for PromptReco

The Tier0 has very unreliable timePerEvent and sizePerEvent estimates as running conditions can change greatly, leading to huge differences within the same dataset from run to run. To not create jobs that can never run, cap the wall time and disk estimates.

hufnagel commented:
@amaltaro , @ticoann , can you review ?

ticoann commented:
retest this please

ticoann commented:
Patch looks good and it didn't seems to be affect if the parameters are not set. Merging it.
